config.yml

```diff
@@ -75,10 +78,63 @@ DoPosterior: false
 DoPreview: true
 DOFSThreshold: 0
 
-## ======================================================================
-## Specific to Harvard's Cannon cluster
-## ======================================================================
+##------------------------------------------------------------------
+## Settings for GEOS-Chem simulations
+##------------------------------------------------------------------
+## Jacobian settings
+PerturbValue: 1.5
+
+## Apply scale factors from a previous inversion?
+UseEmisSF: false
+UseSeparateWetlandSF: false
+UseOHSF: false
+
+## Save out hourly diagnostics from GEOS-Chem?
+## For use in satellite operators via post-processing -- required for TROPOMI
+## inversions
+HourlyCH4: true
+
+## Turn on planeflight diagnostic in GEOS-Chem?
+## For use in comparing GEOS-Chem against planeflight data. The path
+## to those data must be specified in input.geos.
+PLANEFLIGHT: false
+
+## Turn on old observation operators in GEOS-Chem?
+## These will save out text files comparing GEOS-Chem to observations, but have
+## to be manually incorporated into the IMI
+GOSAT: false
+TCCON: false
+AIRS: false
+
+##------------------------------------------------------------------
+## Settings for running on local cluster
+##------------------------------------------------------------------
+
+## Path for IMI runs and output
+OutputPath: "/home/ubuntu/imi_output_dir"
+
+## Path to GEOS-Chem input data
+DataPath: "/home/ubuntu/ExtData"
+
+## Environment files
+source /home/ubuntu/miniconda/etc/profile.d/conda.sh
+CondaEnv: "geo"
+
+## Download initial restart file from AWS S3?
+RestartDownload: true
+
+## Path to initial GEOS-Chem restart file
+RestartFile: "${DataPath}/BoundaryConditions/GEOSChem.BoundaryConditions.${SpinupStart}_0000z.nc4"
+RestartFilePreview: "${DataPath}/BoundaryConditions/GEOSChem.BoundaryConditions.${StartDate}_0000z.nc4"
```

Comment:
Do bash variables get resolved by the `parse_yaml` function? eg. `${DataPath}`

Reply:
Good point. I can update that to include the full path.